seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -958,6 +958,7 @@
       "tests": [
         "assert(new RegExp(\"googleapis\", \"gi\").test(code), 'message: Import the <code>Lobster</code> font.');",
         "assert($(\"h2\").css(\"font-family\").match(/lobster/i), 'message: Your <code>h2</code> element should use the font <code>Lobster</code>.');",
+        "assert(/\\s*?h2\\s*?\\{\\s*?font-family\\:\\s*?(\\'|\\\")?Lobster(\\'|\\\")?\\s*?\\;\\s*?\\}/gi.test(code), 'message: Use an <code>h2</code> CSS selector to change the font.');",
```

Comment:
I fixed the semicolon, fixed the double quotes, and removed the question mark from the whitespaces. I left the double escape on the single quotes because otherwise they show as errors in the editor. Part of the linting maybe?